packages/consensus/src/types/utxo_set.cairo

```diff
@@ -50,3 +59,170 @@ pub impl UtxoSetImpl of UtxoSetTrait {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{UtxoSet, UtxoSetTrait, OutPoint};
+    use consensus::types::transaction::TxOut;
+
+    use core::hash::{HashStateTrait, HashStateExTrait};
+    use core::poseidon::PoseidonTrait;
+
+    #[test]
+    /// To check the validity of expected fields, there is a python program from ZeroSync
+    /// https://github.com/ZeroSync/ZeroSync/blob/main/src/utxo_set/bridge_node.py
+    /// $ python scripts/data/utreexo.py
+    fn test_utreexo_add1() {
```

Comment:
This is a unit test for utreexo which operates on already hashed outputs.
It's better to move it to the utreexo.cairo and leave just outpoint hashes (no need to compute them)